src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -291,6 +291,24 @@ export default class StandardRouteModel extends BaseModel {
     }
 
     /**
+     * Returns an array of fix names used by any portion of the procedure
+     *
+     * @for StandardRouteModel
+     * @method getAllFixNames
+     * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
+     */
+    getAllFixNames() {
```

Comment:
I have thoughts on this. It I need a computer to show you. In the mean time, why not `_flatten()` for the draw section?